core/src/test/java/com/expediagroup/streamplatform/streamregistry/core/services/ConsumerServiceTest.java

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
No delete test here?

Reply:
Seems like you're missing a few unit tests for the delete logic from the services package.